sdk/test/test.go

```diff
@@ -11,8 +12,36 @@ import (
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/bundle"
 	"github.com/open-policy-agent/opa/compile"
+	"github.com/open-policy-agent/opa/plugins"
 )
 
+// Plugin creates an empty plugin to test plugin initialization
```

Comment:
Nit: This file provides utilities for people that want to build integration tests for their code that relies on the SDK. As such, it's not the right place for this mock plugin implementation which appears to serve the test in opa_test.go. Just move this over to the opa_test.go file and unexport the plugin and factory implementations.